statics/js/components/layout.js

```diff
@@ -1,38 +1,39 @@
-var hostImg = 'statics/img/host.png';
-var switchImg = 'statics/img/switch.png';
-var portImg = 'statics/img/port.png';
-var intfImg = 'statics/img/intf.png';
-var vethImg = 'statics/img/veth.png';
-var nsImg = 'statics/img/ns.png';
-var bridgeImg = 'statics/img/bridge.png';
-var dockerImg = 'statics/img/docker.png';
-var podImg = 'statics/img/pod.png';
-var networkPolicyImg = 'statics/img/networkpolicy.png';
-var neutronImg = 'statics/img/openstack.png';
-var k8sImg = 'statics/img/k8s.png';
+var getImageFile = function(name) {
+	return 'statics/img/'+name+'.png'
+}
 
-var minusImg = 'statics/img/minus-outline-16.png';
-var plusImg = 'statics/img/plus-16.png';
-var captureIndicatorImg = 'statics/img/media-record.png';
-var pinIndicatorImg = 'statics/img/pin.png';
+var minusImg = getImagePath('minus-outline-16');
+var plusImg = getImagePath('plus-16');
+var captureIndicatorImg = getImagePath('media-record');
+var pinIndicatorImg = getImagePath('pin');
 
-var nodeImgMap = {
-  "host": hostImg,
-  "port": portImg,
-  "ovsport": portImg,
-  "bridge": bridgeImg,
-  "switch": switchImg,
-  "ovsbridge": switchImg,
-  "netns": nsImg,
-  "veth": vethImg,
-  "bond": portImg,
-  "container": dockerImg,
-  "pod": podImg,
-  "networkpolicy": networkPolicyImg
+var getNodeImage = function(type) {
+  var nodeImgMap = {
```

Comment:
As said before it is now impossible for a user to customize the UI without rewritten the whole function. The goal here was to allow user to add their own image to the map thanks to the config file.